### DISCUSSION TITLE
Cartoon Network India

### DISCUSSION THREAD

**Discussion by Achu2234** [iptv-org/community#786] [Q&A]
### Channel Name

Cartoon Network India

### Channel ID

CartoonNetwork.in

### Website

https://www.cartoonnetworkindia.com/

### Notes

I have searching it for long plx give me

Language : hindi

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
The channel has been added to our blocklist due to the claims of the copyright holder: [#16839](https://github.com/iptv-org/iptv/issues/16839)
